docs/pages/machine-id/access-guides/ansible.mdx

```diff
@@ -31,7 +31,75 @@ You will need the following tools to use Teleport with Ansible.
 
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/2. Configure Ansible
+## Step 1/4. Configure RBAC
+
+First, Teleport must be configured to allow the credentials produced by the bot
+to connect to the SSH servers. In this example, access will be granted to all
+SSH hosts for the username `root`.
+
+For production use, you should use labels to restrict this access to only the
+hosts that Ansible will need to access. This is known as the principal of
+least privilege and reduces damage that exfiltrated credentials can do.
+
+If you have followed an access guide, you will have created a role and granted
+the bot the ability to impersonate it already. This role just needs to have the
+additional privileges added to it.
+
+Use `tctl edit role/example-bot` to add the following to the role:
+
+```yaml
+spec:
+  allow:
+    # Allow login to the user 'root'.
+    logins: ['root']
```

Comment:
Since we assign the `remote_user` field of the playbook to a `Var` later on, would it make sense to do this here as well? We could also instruct the user to replace `root` if there's another user on the host that would work.

Reply:
Addressed in 4ec7753c9f